src/lib/leagues/masterTasks.ts

```diff
@@ -1090,12 +1092,16 @@ export const masterTasks: Task[] = [
 		}
 	},
 	{
-		id: 4151,
+		id: 4151, // 50 mystery impling, 100 of every other imping (excluding Lucky implings)
 		name: 'Catch 100 of every impling passively (excluding Lucky implings)',
 		has: async ({ userStats }) => {
```

Comment:
If we're adding this extra special case with mystery implings, the title should mention it, not in a comment users cant see.

Reply:
Changed it to "Catch 50 mystery impling and 100 of every other imping passively (excluding Lucky implings)" let me know how that sounds